README.md

```diff
@@ -8,13 +8,21 @@ You will need [Leiningen][] 2.0.0 or above installed.
 
 [leiningen]: https://github.com/technomancy/leiningen
 
+(with Clojure 1.7.0+)
+
+Data Store: Cassandra
+
+Example Configuration in <https://github.com/MastodonC/witan.app/blob/master/resources/dev.witan-app.edn>: this file is also the default. If necessary, copy to home under .witan-app.edn and modify to suit.
+
 ## Running
 
 To start a web server for the application, run:
 
     lein ring server
```

Comment:
was actually default doc (from lein template :) )
for the beginning user it's not bad to have the docs opening automatically.